controller/tests/controller_server/utils_test.py

```diff
@@ -351,14 +351,37 @@ def test_validate_unpublish_volume_request(self, validate_secrets):
 
         utils.validate_unpublish_volume_request(request)
 
+    def _test_get_volume_id_info(self, object_id, system_id=None, internal_id=None):
+        system_id_field = ':' + system_id if system_id else ''
+        id_field = internal_id + ';' + object_id if internal_id else object_id
```

Comment:
I think `ids_field` would be better here

Reply:
done